packages/Webkul/Shop/src/Routes/api.php

```diff
@@ -103,4 +104,7 @@
             Route::delete('{id}', 'destroy')->name('shop.api.customers.account.wishlist.destroy');
         });
     });
+
+    // for formatting total price in bundle product
+    Route::get('/formatPrice/{slug1}/{slug2}/', [Core::class, 'formatPrice'])->name('shop.api.format_price');
```

Comment:
Don't add the core class directly. Use controller.

Reply:
Done.